src/components/ClassCards.js

```diff
@@ -39,7 +39,7 @@ const ClassCards = ({ activeLevels, slugExtension = "" }) => {
                     imageInfo={{
                       image: thumbnail,
                       alt: title,
-                      imageStyle: { width: "100%", height: "auto" },
+                      imageStyle: { width: "100%", height: "200px" },
```

Comment:
does this contain something like an `object-fit: cover;`

https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit

Reply:
Let me double-check it. There's an `object-fit: "contain"` on png images but not others. Might have just gotten lucky with the images I had because they all look alright (not distorted)

Reply:
Look's like yes,  GatsbyImg injects `position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center center; opacity: 1; transition: opacity 500ms ease 0s;`

Reply:
Nice!

Just making sure. The link I sent should help clarify why setting that is beneficial when working to standardize non-standard images (basically scales from the center instead of top left)